docs/shocks.rst

```diff
@@ -1,140 +1,134 @@
 
-Configuring the simulation
-==========================
-To configure the simulation, you can edit the `shock_mini.ini` or `shock_small.ini` files given. The smaller of those (mini) should finish in less than a minute whereas the small takes ~20mins to finish.
+**Example:** Running a simulation with 4 cores using the configuration file ``2dsig3.ini``: 
 
-The following settings are given:
+.. code-block:: bash
 
-- [io]
-   - `outdir`: Defines the output directory name
-   - `interval`: No. of steps between output files written
-   - `restart`: No. of steps between restart files written
-   - `laprestart`: Simulation lap to restart from (-1 = no restart, 0 = automatic, X lap to restart)
+    mpirun -n 4 python3 pic.py --conf 2dsig3.ini
+    
 
-Additionally, more advanced options include
+Configuration Parameters
+++++++++++++++++++++++++
 
-   - `stride`: Output image reduce factor
-   - `full_interval`: No. of steps between full restart snapshot written to file (-1 = disabled)
 
-- [simulation]
-   - `cfl`: The simulation time step in units of grid space
-   - `Nt`: The total number of laps in the simulation
-   
-- [problem]
-   - `Nspecies`: Number of species
-   - `delgam`: Temperature of plasma in units of mc^2 in bulk flow frame
-   - `gamma`: Lorentz factor of bulk flow
-   - `me`: electron mass-to-charge
-   - `mi`: positron/ion mass-to-charge
-   - `sigma`: Plasma magnetization parameter
-   - `npasses`: Number of current filter passes
-   
-Additionally, more advanced options include
+Grid and Dimensions
+------------------------
 
-   - `temp_ratio`: Ratio of ion temperature to electron temperature
-   - `bphi`: external B-field z angle
-   - `btheta`: external B-field x-y angle
-   - `wallgamma`: X velocity of the left piston wall (this makes the left wall ram into the plasma)
+Simulations are available in all three dimensions and you can set the size of the grid based on your needs. The following configuration parameters are found and can be edited in the ``.ini`` files.
 
-- [grid]
-   - `Nx`, `Ny`, `Nz`: No. of mesh tiles in x,y,z direction
-   - `NxMesh`/`NyMesh`/`NzMesh`: Size of tiles in x,y,z direction
-   
-Complete grid size is then `Ni*NiMesh`.
+Simulation framework:
 
-- [particles]
-   - `ppc`: Particles per cell per species
-   - `c_omp`: Skindepth resolution
+- ``[grid]``
+   - ``Nx``, ``Ny``, ``Nx`` : Number of tiles in corresponding directions
+   - ``NxMesh``, ``NyMesh``, ``NzMesh`` : Number of partitions each tile is divided into in each corresponding direction; internal grid.
+   - ``oneD``, ``twoD``, ``threeD`` : Forced dimensionality - `True` forces the dimension when other options are set to `False`.
+   - ``c_omp`` : Simulation skin depth resolution
 
-Additionally, more advanced options include
+Complete grid size in each dimension is determined as ``Ni*NiMesh``.
```

Comment:
We could add here more explanations for the user:

The complete grid size in each dimension is determined as ``Ni*NiMesh``. One plasma skin depth (in the upstream) equals ``c_omp`` `\times` cells.